### ISSUE TITLE
When using guidelines, show centre arrows and grid counters

### ISSUE THREAD

**Issue by glasnt** [glasnt/ih#3]
Based on https://twitter.com/drsarahlgibson/status/1229061090560049157

 * have centre arrows
 * ~no-colour mode (for printing)~

**Comment by glasnt** [comment]
* printing mode shows black/white.

**Comment by glasnt** [comment]
On centre arrows: 

Some of the patterns I've purchased lately from other artists include lovely things like the centre grid line being brighter, as well as big arrows, and numbers at 10 intervals

<img width="78" alt="Example gridline" src="https://user-images.githubusercontent.com/813732/111060404-b34b3500-84f0-11eb-8d6a-c031ba3d38e2.png"> | <img width="78" alt="ANother example gridline" src="https://user-images.githubusercontent.com/813732/111060417-d249c700-84f0-11eb-81ef-5f6fe8ae51e2.png"> | <img width="72" alt="and another" src="https://user-images.githubusercontent.com/813732/111060426-e5f52d80-84f0-11eb-8dbf-eb5046c88bed.png">

**Comment by rlaker** [comment]
I agree with this suggestion, it is hard to find the centre pixel when the stitching is red!
